lib/features/dataset/display.dart

```diff
@@ -74,6 +78,10 @@ class _DatasetDisplayState extends ConsumerState<DatasetDisplay> {
   final int typeFlex = 4;
   final int contentFlex = 3;
 
+  // Track pressed keys for shift selection
+
+  bool _isShiftPressed = false;
```

Comment:
Would the DataTable widget be better for supporting selecting rows in a standard way?

Reply:
will solve this in https://github.com/gjwgit/rattleng/issues/581